client/my-sites/plugins/plugins-list/index.tsx

```diff
@@ -532,14 +575,18 @@ export class PluginsList extends Component {
 		const { plugins } = this.props;
 		return plugins.reduce( ( sites, plugin ) => {
 			Object.keys( plugin.sites ).map( ( pluginSiteId ) => {
-				if ( ! sites.find( ( site ) => site.ID === pluginSiteId ) ) {
-					const pluginSite = this.props.allSites.find( ( s ) => s.ID === parseInt( pluginSiteId ) );
-					sites.push( pluginSite );
+				if ( ! sites.find( ( site ) => site && site.ID === Number( pluginSiteId ) ) ) {
+					const pluginSite = this.props.allSites.find(
+						( s ) => s && s.ID === parseInt( pluginSiteId )
```

Comment:
Any good reason to prefer `parseInt()` to `Number()` for casting to a number (L578 vs L580)? Seems like we can try to be consistent about the one we prefer. In any case, from my understanding they'll work the same way when working with a string that contains a number.

Reply:
In this case it was just pre-existing code that I failed to recognize. I switched it to `Number` for consistency.